src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -2267,7 +2324,28 @@ public static PartitionIterator readRegular(SinglePartitionReadCommand.Group gro
         long start = nanoTime();
         try
         {
-            PartitionIterator result = fetchRows(group.queries, consistencyLevel, coordinator, requestTime);
+            ClusterMetadata cm = ClusterMetadata.current();
+            TableId tableId = group.queries.get(0).metadata().id;
+            // Returns null for local tables
+            TableMetadata tableMetadata = getTableMetadata(cm, tableId);
+            if (tableMetadata == null)
+                tableMetadata = Schema.instance.localKeyspaces().getTableOrViewNullable(tableId);
+            TableParams tableParams = tableMetadata.params;
+
+            TransactionalMode transactionalMode = tableParams.transactionalMode;
+            TransactionalMigrationFromMode transactionalMigrationFromMode = tableParams.transactionalMigrationFrom;
+            if (transactionalMigrationFromMode != TransactionalMigrationFromMode.none && transactionalMode.readsThroughAccord && transactionalMigrationFromMode.writesThroughAccord() && transactionalMigrationFromMode.readsThroughAccord())
```

Comment:
what case is this?  if the read/writes *from* are accord... then are we migrating from `full` to `full`?

Reply:
I think this is just 100% wrong, and I'll also bet that if this thing actually worked it would have made a bunch of the tests fail when we test live migration to full. 

We don't have the looping and splitting to handle races and route different parts of the query to Accord/Cassandra so really no migration should be allowed.

I think for today we should just remove this check.